### ISSUE TITLE
Confusing LEF/DEF import behaviour with symlinks

### ISSUE THREAD

**Issue by antonblanchard** [KLayout/klayout#965]
If I import a DEF file that is a symlink, the import window shows the LEF files in the same directory as the symlink (making out as if it is going to use them), but it doesn't, doesn't warn that it did not open them, and ultimately fails.

I'm guessing klayout follows the symlink and then looks for LEF files there (based on strace output).

Another point of confusion, it appears that the list of LEF files is not used and it just reads every LEF in the same directory as the DEF 
regardless of the contents.

Testcase: [lefdef-import.tar.gz](https://github.com/KLayout/klayout/files/7819041/lefdef-import.tar.gz)

**Comment by klayoutmatthias** [comment]
Thanks for mentioning the symlink topic.

Are you using the "import LEF/DEF" feature? This is kind of redundant because as of now you can simply read DEF like the other layout formats. This function will be removed soon.

The primary way to configure the DEF reader is through "File/Reader Options". When you're using the "Import" function, the import parameters can be configured with the "Options" button in the import dialog.

Here is actually the place where to add LEF files. And here is also the explanation for the "Reads all LEF files" problem:

![image](https://user-images.githubusercontent.com/25720841/149637310-388a8515-5640-4de2-a97f-17e0a1da3ca5.png)

In this case the "Read all LEF files in the same directory" is checked. Uncheck this option if you don't want to have this.

Basically, the LEF entries are intended for some technology setup specific LEF files (tech LEF for example). If you're working with technologies, these LEF files are resolved relative to the technology directory and you can store tech specific or PDK LEF files there.

Matthias

**Comment by klayoutmatthias** [comment]
Is this issue solved? I'd like to close it.

**Comment by antonblanchard** [comment]
Thanks @klayoutmatthias I was indeed using the old LEF/DEF import.
